app/src/components/disclosure/styles.tsx

```diff
@@ -0,0 +1,74 @@
+import { css } from "@emotion/react";
+
+export const disclosureGroupCss = css`
+  & > * {
+    width: 100%;
+    .react-aria-Heading {
+      width: 100%;
+      .react-aria-Button[slot="trigger"] {
+        width: 100%;
+      }
+    }
+  }
+`;
```

Comment:
I'm thinking of just putting this on the disclosure itself and not having any styles for the group.

Right now you have to wrap a single Disclosure in a group if you want it to fill its container, otherwise its only as long as its title.

Either that, or I add a width prop to Disclosure